src/cpu.rs

```diff
@@ -42,6 +44,14 @@ bitflags! {
     }
 }
 
+#[cfg(any(target_arch = "riscv64", target_arch = "riscv32"))]
+bitflags! {
+    #[derive(Clone, Copy)]
+    pub struct CpuFlags: c_uint {
+        const V = 1 << 0;
+    }
+}
```

Comment:
We should `git cherry-pick` the corresponding `dav1d` commits for `riscv` asm support so that our C stays in sync.

Reply:
Done